Dockerfile

```diff
@@ -138,13 +151,15 @@ ENV HZ_CLIENT_MODE=false \
 RUN  mkdir -p /opt/pega/kafkadata && \
      chgrp -R 0 /opt/pega/kafkadata && \
      chmod -R g+rw /opt/pega/kafkadata && \
+     chmod -R o+rwx /opt/pega/kafkadata && \
      chown -R pegauser /opt/pega/kafkadata
```

Comment:
Is the desired end state of kafkadata is pegauser:root 766?  Weren't we seeing, kafkadata being mounted in as root:root 640?  We should probably try to be consistent between the Dockerfile and the entrypoint.sh.   Perhaps not bother setting permissions, or ownerships in the Dockerfile and manage it entirely in entrypoint.sh?  Otherwise it might lead to confusion.

Reply:
just change to 777, openshift will run as root group, and will need to create the dirs as well